pyproject.toml

```diff
@@ -32,6 +32,7 @@ requires-python = ">=3.10"
 dependencies = [
     "scipp >= 24.02.0",
     "scippnexus >= 24.03.0",
+    "scitacean[sftp, test]",
```

Comment:
If we avoid using extras then the new way of defining `meta.yaml` automatically (see `copier_template`) will work. Also, why do we depend on the `test` extra, that seems odd?

Reply:
Yes I'll remove the `test` extra. How can we avoid depending on the `sftp` extra? By manually adding the dependencies?